pyblish_lite/control.py

```diff
@@ -95,6 +95,10 @@ def on_next():
 
         util.defer(100, on_next)
 
+    def controlEmit(self, signal, kwargs):
```

Comment:
Ah, I see why you needed to avoid calling it `emit`. For some reason, `QObject` has a method with that name already. I think let's bite the bullet and give it a name that more explicitly conveys that we're interested in a duplicate, `emit_`. An underscore suffix is conventional for those things. At least until we find a better way.